components/feedstock/thumbnail.js

```diff
@@ -1,8 +1,38 @@
 import { Badge } from '@carbonplan/components'
 import { alpha } from '@theme-ui/color'
-import { Box } from 'theme-ui'
+import { Box, Flex } from 'theme-ui'
 
-export const Thumbnail = ({ url, color, demo }) => {
+const Tags = ({ tags, sx }) => {
+  // Sort tags alphabetically
+  const sortedTags = tags.sort((a, b) => a.localeCompare(b))
+  return (
+    <Flex
+      sx={{
+        columnGap: 2,
+        rowGap: 1,
+        flexWrap: 'wrap',
+        ...sx,
+      }}
+    >
+      {sortedTags.map((tag) => (
+        <Badge
+          key={tag}
+          sx={{
+            fontSize: [1],
```

Comment:
Similar callout here! The [`Badge` component styles](https://github.com/carbonplan/components/blob/main/src/badge.js) are defined responsively, so it's important that any changes to size/spacing are tested responsively as well. 

<img width="364" alt="CleanShot 2024-04-18 at 10 07 41@2x" src="https://github.com/carbonplan/leap-data-catalog/assets/12436887/611772c8-2fe9-4e18-971b-ec93ad0d7538">